peazip-actions@BlancoBv/files/peazip-actions@BlancoBv/metadata.json

```diff
@@ -0,0 +1,7 @@
+{
+    "description": "Integrate peazip actions to contextual menu",
+    "uuid": "peazip-actions@100savage",
```

Comment:
This is incorrect. (It should have failed validation so I'll look into that separately.)